conda_smithy/ci_register.py

```diff
@@ -214,13 +214,19 @@ def travis_wait_until_synced(ignore=False):
     return content
 
 
-def travis_get_repo_info(user, project, show_error=False):
```

Comment:
Why change the function name in this PR?
Personally, I feel like the "info" part captures the fact that there is no such thing as a "repo" object...